server/src/main/java/io/druid/curator/CuratorConfig.java

```diff
@@ -66,4 +69,14 @@ public void setEnableCompression(Boolean enableCompression)
   {
     this.enableCompression = enableCompression;
   }
+
+  public boolean getEnableAcl()
+  {
+    return enableAcl;
+  }
+
+  public void setEnableAcl(Boolean enableAcl)
+  {
+    this.enableAcl = enableAcl;
```

Comment:
I agree with @fjy on this one. But would still like to see `Preconditions.notNull(enableAcl, "enableAcl")` here